drivers/ethernet/eth_native_posix.c

```diff
@@ -559,6 +559,12 @@ static int set_config(const struct device *dev,
 
```

Comment:
Please change the commit subject to
`drivers: eth: native_posix: Add support for setting ethernet MAC address`
so that this commit does not get confused to Posix subsystem.

Reply:
Will do.